apps/modernization-ui/src/pages/patient/profile/identification/IdentificationsTable.tsx

```diff
@@ -58,38 +57,70 @@ type Props = {
     patient: Patient | undefined;
 };
 
+const headers = [
+    { name: Column.AsOf, sortable: true },
+    { name: Column.Type, sortable: true },
+    { name: Column.Authority, sortable: true },
+    { name: Column.Value, sortable: true },
+    { name: Column.Actions, sortable: false }
+];
+
 export const IdentificationsTable = ({ patient }: Props) => {
     const { showAlert } = useAlert();
-    const [tableHead, setTableHead] = useState<{ name: string; sortable: boolean; sort?: string }[]>([
-        { name: 'As of', sortable: true, sort: 'all' },
-        { name: 'Type', sortable: true, sort: 'all' },
-        { name: 'Authority', sortable: true, sort: 'all' },
-        { name: 'Value', sortable: true, sort: 'all' },
-        { name: 'Actions', sortable: false }
-    ]);
+    const [bodies, setBodies] = useState<TableBody[]>([]);
+    const [items, setItems] = useState<Identification[]>([]);
 
     const [total, setTotal] = useState<number>(0);
     const [currentPage, setCurrentPage] = useState<number>(1);
 
     const initial = resolveInitialEntry(patient?.id || '');
     const { changed } = useProfileContext();
 
-    const [isActions, setIsActions] = useState<any>(null);
-    const [identifications, setIdentifications] = useState<PatientIdentification[]>([]);
+    const [actionIndex, setActionIndex] = useState<number | null>(null);
+
+    const asTableBodies = (idenitifications: Identification[]): TableBody[] =>
+        idenitifications?.map((identification, index) => ({
+            id: identification.value,
+            tableDetails: [
+                {
+                    id: 1,
+                    title: format(identification.asOf, 'MM/dd/yyyy')
```

Comment:
The old code shows a <NoData /> when api doesn't provide asOf. I think we missed putting that in

Reply:
okay will check